db/schema.yml

```diff
@@ -6459,6 +6459,9 @@ system_consoles:
 - user_id
 - created_at
 - updated_at
+- proxy_pid
+- url
+- proxy_status
```

Comment:
I'm not sure how your columns can be in this order if the migration is in url, pid, status order.